frontend/src/components/Header.js

```diff
@@ -22,19 +23,27 @@ const Header = () => {
             </ApplicationNameContainer>
             <NavigationBar>
                 <NavItem to="/">
-                    <NavLogoContainer src={process.env.PUBLIC_URL + "/NavLogoHome.svg"} alt={"HomeLogo"}/>
+                    <ImageWrapper>
+                        <Image src={process.env.PUBLIC_URL + "/HomeLogo.svg"} alt={"HomeLogo"}/>
+                    </ImageWrapper>
                     <NavTitle>Home</NavTitle>
                 </NavItem>
                 <NavItem to="/search-inputs" >
-                    <NavLogoContainer src={process.env.PUBLIC_URL + "/NavLogoSearchInputs.svg"} alt={"SearchInputsLogo"}/>
+                    <ImageWrapper>
+                        <Image src={process.env.PUBLIC_URL + "/DetailsLogo.svg"} alt={"SearchInputsLogo"}/>
```

Comment:
usually React knows to look in the public folder without declaring a  PUBLIC_URL
also the alt should be a descriptive text for screen readers etc. and in standad text format instead of PascalCasl
```suggestion
                        <Image src={"/DetailsLogo.svg"} alt={"Search Inputs Logo"}/>
```